### PR TITLE
implement wait_for_thrift_interface

### PR DESCRIPTION
**Pull request by smccarthy788**
@ptnapoleon & @mambocab, please review.

There is one thing I'm not sure about, which I had just copied over from `wait_for_binary_interface()`: the conditional based on version number 1.2 in line 508. Is this the correct version?

### CODE REVIEW

File: ccmlib/node.py [hunk 1]
```diff
@@ -497,6 +497,21 @@ def wait_for_binary_interface(self, **kwargs):
             warnings.warn("Binary interface %s:%s is not listening after 10 seconds, node may have failed to start."
                           % (binary_itf[0], binary_itf[1]))
 
+    def wait_for_thrift_interface(self, **kwargs):
+        """
+        Waits for the Thrift interface to be listening.  If > 1.2 will check
+        log for 'Starting listening for CQL clients' before checking for the
```
Comment (ptnapoleon) [inline ccmlib/node.py]: This comment is now unrelated. Just keep the first sentence of this paragraph.

File: ccmlib/node.py [hunk 2]
```diff
@@ -497,6 +497,21 @@ def wait_for_binary_interface(self, **kwargs):
...
+        interface to be listening.
+
+        Emits a warning if not listening after 10 seconds.
+        """
+        if self.cluster.version() >= '1.2':
```
Comment (ptnapoleon) [inline ccmlib/node.py]: Drop the `if` check

**Comment by smccarthy788** [comment]
@ptnapoleon, I have addressed your review comments.

File: ccmlib/node.py (hunk 2 above)
Comment (ptnapoleon) [inline ccmlib/node.py]: My bad, I meant only drop the if check, not the code inside the if check.

**Comment by ptnapoleon** [comment]
+1

**Pull request merged**